fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/LinkHeaderProvider.java

```diff
@@ -66,15 +66,15 @@
     public Multimap<String, String> createHttpHeadersForResource(final UriInfo uriInfo, final FedoraResource resource) {
 
         // TODO do not use transactions for internal reads
-        final Transaction transaction = sessionFactory.getNewTransaction();
+        final Transaction transaction = txManager.create();
```

Comment:
looking at this again, we probably won't want to be creating a new transaction here, I'd think that we'd want to use the already active transaction from the request. Would it be possible to use the TransactionProvider here to get it?

Reply:
I will refactor this PR to no use transaction once #1561 is merged.

Reply:
Updated to existing transaction associated with the request.